core/src/main/scala/org/http4s/MimeDB.scala

```diff
@@ -0,0 +1,6700 @@
+      `text/calendar`,
+      `text/calender`,
+      `text/cmd`,
+      `text/coffeescript`,
+      `text/css`,
+      `text/csv`,
+      `text/csv-schema`,
+      `text/directory`,
+      `text/dns`,
+      `text/ecmascript`,
+      `text/encaprtp`,
+      `text/enriched`,
+      `text/fwdred`,
+      `text/grammar-ref-list`,
+      `text/html`,
+      `text/jade`,
+      `text/javascript`,
+      `text/jcr-cnd`,
+      `text/jsx`,
+      `text/less`,
+      `text/markdown`,
+      `text/mathml`,
+      `text/mizar`,
+      `text/n3`,
+      `text/parameters`,
+      `text/parityfec`,
+      `text/plain`,
+      `text/provenance-notation`,
+      `text/prs.fallenstein.rst`,
+      `text/prs.lines.tag`,
+      `text/prs.prop.logic`,
+      `text/raptorfec`,
+      `text/red`,
+      `text/rfc822-headers`,
+      `text/richtext`,
+      `text/rtf`,
+      `text/rtp-enc-aescm128`,
+      `text/rtploopback`,
+      `text/rtx`,
+      `text/sgml`,
+      `text/shex`,
+      `text/slim`,
+      `text/strings`,
+      `text/stylus`,
+      `text/t140`,
+      `text/tab-separated-values`,
+      `text/troff`,
+      `text/turtle`,
+      `text/ulpfec`,
+      `text/uri-list`,
+      `text/vcard`,
+      `text/vnd.a`,
+      `text/vnd.abc`,
+      `text/vnd.ascii-art`,
+      `text/vnd.curl`,
+      `text/vnd.curl.dcurl`,
+      `text/vnd.curl.mcurl`,
+      `text/vnd.curl.scurl`,
+      `text/vnd.debian.copyright`,
+      `text/vnd.dmclientscript`,
+      `text/vnd.dvb.subtitle`,
+      `text/vnd.esmertec.theme-descriptor`,
+      `text/vnd.fly`,
+      `text/vnd.fmi.flexstor`,
+      `text/vnd.graphviz`,
+      `text/vnd.in3d.3dml`,
+      `text/vnd.in3d.spot`,
+      `text/vnd.iptc.newsml`,
+      `text/vnd.iptc.nitf`,
+      `text/vnd.latex-z`,
+      `text/vnd.motorola.reflex`,
+      `text/vnd.ms-mediapackage`,
+      `text/vnd.net2phone.commcenter.command`,
+      `text/vnd.radisys.msml-basic-layout`,
+      `text/vnd.si.uricatalogue`,
+      `text/vnd.sun.j2me.app-descriptor`,
+      `text/vnd.trolltech.linguist`,
+      `text/vnd.wap.si`,
+      `text/vnd.wap.sl`,
+      `text/vnd.wap.wml`,
+      `text/vnd.wap.wmlscript`,
+      `text/vtt`,
+      `text/x-asm`,
+      `text/x-c`,
+      `text/x-component`,
+      `text/x-fortran`,
+      `text/x-gwt-rpc`,
+      `text/x-handlebars-template`,
+      `text/x-java-source`,
+      `text/x-jquery-tmpl`,
+      `text/x-lua`,
+      `text/x-markdown`,
+      `text/x-nfo`,
+      `text/x-opml`,
+      `text/x-org`,
+      `text/x-pascal`,
+      `text/x-processing`,
+      `text/x-sass`,
+      `text/x-scss`,
+      `text/x-setext`,
+      `text/x-sfv`,
+      `text/x-suse-ymp`,
+      `text/x-uuencode`,
+      `text/x-vcalendar`,
+      `text/x-vcard`,
+      `text/xml`,
+      `text/xml-external-parsed-entity`,
+      `text/yaml`
+    )
+  }
+  object application {
+    lazy val all
+      : List[MediaType] = Nil ::: application.all ::: application_1.all ::: application_2.all
```

Comment:
This is slightly convoluted. Given that there are several thousand `application/` mime types the classes get too large and we hit the JVM 64kb limit. Splitting them this way lets us workaround that error